index.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+$db = new PDO('mysql:host=db; dbname=collections_display', 'root', 'password');
+
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+
+$query = $db->prepare('SELECT * FROM `board_games`;');
+
+$query->execute();
+
+$result = $query->fetchAll();
```

Comment:
As far as brining in classes, you should start with this bit of code.

This should belong to a model class, called something like GamesModel

The job of the GamesModel class is to handle/contain all of the database queries that you need for the board_games table. Rather than having queries scattered around your code, they are all organised into a single class which we would call a model.

Have a look at your examples of a model, or the ones that I sent over (i think for Products) and see how it's just a bunch of DB queries organised into a single class.

Reply:
I have now done this with the above code. I will look to also to add another model class to hold the queries from my 2nd web page although I am going to commit what I currently have shortly.